Sources/Recycling/ListViews/PaymentOptions/Cell/PaymentOptionsListViewCell.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public class PaymentOptionsListViewCell: UITableViewCell {
+    // MARK: - Private properties
+
+    private lazy var selectionBox: SelectionboxItem = {
+        let animatedImageView = AnimatedRadioButtonView(frame: .zero)
+        let selectionBox = SelectionboxItem(index: 0, animatedImageView: animatedImageView)
+        selectionBox.translatesAutoresizingMaskIntoConstraints = false
+        return selectionBox
+    }()
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel(withAutoLayout: true)
```

Comment:
Any reason for not using `Label` in these labels?

Reply:
Totally forgot, will fix.